edgedb/protocol/protocol.pyx

```diff
@@ -1213,10 +1079,10 @@ cdef class SansIOProtocol:
 
     cdef parse_command_complete_message(self):
         assert self.buffer.get_message_type() == COMMAND_COMPLETE_MSG
-        attrs = self.parse_headers()
+        self.parse_headers()
```

Comment:
Should probably rename to ignore_headers

Reply:
I'll re-implement `parse_headers()` in the new way with argument `ignore: bool`.